## CI Failure Analysis — commit 3ccb826

**Total failures**: 1 (1 infrastructure)

---

<details>
<summary>🔧 Infrastructure: <code>pull-cluster-network-addons-operator-unit-test</code> — Container image download failed with "unexpected EOF" while pulling ubi8/ubi-minimal from registry.access.redhat.com/Quay.io CDN</summary>

### Error
````
```
[2/2] STEP 1/9: FROM --platform=linux/arm64 registry.access.redhat.com/ubi8/ubi-minimal
Copying blob sha256:2879d533a3ed260f6964c746c5cdcb795f4b9f34ebd6c79807cc04faf5045789
Error: creating build container: unable to copy from source docker://registry.access.redhat.com/ubi8/ubi-minimal:latest: copying system image from manifest list: writing blob: storing blob to file "/var/tmp/container_images_storage1103519521/1": happened during read: unexpected EOF (while reconnecting: Get "https://cdn01.quay.io/quayio-production-s3/sha256/28/2879d533a3ed260f6964c746c5cdcb795f4b9f34ebd6c79807cc04faf5045789?...": EOF)
+ rm -rf /tmp/deploy.linux-bridge.a2dP
make[1]: *** [Makefile:246: bump-linux-bridge] Error 125
make: *** [Makefile:248: bump-all] Error 2
```
````

### Root Cause
The CI job failed due to a network/infrastructure issue when downloading the ubi8/ubi-minimal container image from the Red Hat container registry backed by Quay.io CDN. The error "unexpected EOF" while reconnecting to the CDN indicates a network timeout or connection drop during the blob download, which is a transient infrastructure failure unrelated to code changes.

### Recommendation
Retest with /retest. This is a transient CDN/network issue that should resolve on retry as the container registry becomes available again.

</details>

<!-- oompa-bot ci:3ccb826:pull-cluster-network-addons-operator-unit-test -->

--- a/hack/deploy-kubevirt.sh
+++ b/hack/deploy-kubevirt.sh
@@ -2,8 +2,7 @@
 set -exo pipefail
 
 if [ -z "${KUBEVIRT_VERSION}" ];then
-  # Get latest stable KubeVirt version
-  export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | grep tag_name | grep -v -- - | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
+  export KUBEVIRT_VERSION=$(curl -sSfL https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
 fi
 
 ./cluster/kubectl.sh apply -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml

